ui/pages/confirmations/confirmation/alerts/alerts-context.tsx

```diff
@@ -0,0 +1,89 @@
+import { ApprovalRequest } from '@metamask/approval-controller';
+import React, {
+  ReactElement,
+  createContext,
+  useCallback,
+  useContext,
+  useState,
+} from 'react';
+
+import useAlerts from '../../../../hooks/useAlerts';
+import { AlertActionHandlerProvider } from '../../../../components/app/alert-system/contexts/alertActionHandler';
+import { AlertMetricsProvider } from '../../../../components/app/alert-system/contexts/alertMetricsContext';
+import { ConfirmAlertModal } from '../../../../components/app/alert-system/confirm-alert-modal';
+import setAlerts from './setAlerts';
+
+type AlertContextType = {
+  hasAlerts: boolean;
+  showAlertsModal: () => void;
+};
+
+const NopeFunction = () => undefined;
```

Comment:
Minor, `lodash` also  has a standard function for this called `noop`.

Reply:
This is actually going away in next PR.